OneCommunityGlobal/HGNRest
# Jatin get profile images from website

# Description
<img width="733" alt="Screenshot 2024-09-08 at 1 41 52 PM" src="https://github.com/user-attachments/assets/506d538c-2ed5-4dcb-b2c3-7b1f31463d16">

## Related PRS (if any):
To test this backend PR you need to checkout the [#2714 ](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2714)frontend PR.

## Main changes explained:
- Functions added in `usercontroller.js` for removing and updating profile images.
- userhelper function created in `userhelper.js` for getting images from website.
- New key added to userProfile Collection names `suggestedProfilePics` to store images from website.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3.  Go to server.js and enter the line of code shown in the video as "for testing purposes only",
4. Make changes to the query as stated in the video.
5. Run `npm run build` and `npm start` .
6. Go to frontend repo for next steps.

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/b60da6c0-0375-4fc8-9a5a-e2912c29463f
